pkg/koordlet/metricsadvisor/collectors/podresource/pod_resource_collector.go

```diff
@@ -143,45 +143,58 @@ func (p *podResourceCollector) collectPodResUsed() {
 		// do subtraction and division first to avoid overflow
 		cpuUsageValue := float64(currentCPUUsage-lastCPUStat.CPUUsage) / float64(collectTime.Sub(lastCPUStat.Timestamp))
 
-		memUsageValue := memStat.Usage()
+		cpuUsageMetric, err := metriccache.PodCPUUsageMetric.GenerateSample(
+			metriccache.MetricPropertiesFunc.Pod(uid), collectTime, cpuUsageValue)
+		if err != nil {
+			klog.Warningf("generate pod cpu metrics failed, err %v", err)
```

Comment:
`klog.V(4).Infof()` is preferred. How about `klog.V(4).Infof("failed to generate pod cpu metrics for pod %s, err %v", podKey, err)`? Same below.